### DISCUSSION TITLE
Broadcastchannel Explicit Subscription

### DISCUSSION THREAD

**Discussion by sc-starman** [dotnet/orleans#8477] [Q&A]
I read all the docs and saw all the examples but couldn't find out explicit subscription for broadcast channel
Is there a way to subscribe to a Broadcastchannel for a background service?

**Comment by artemiusgreat** [comment]
Bump. 
Looks like there are other devs interested in how to subscribe to `BroadcastChannel` outside the grain. 
Example below shows only subscription from the grain. 
https://learn.microsoft.com/en-us/dotnet/orleans/streaming/broadcast-channel
